jexl-parser/src/parser.lalrpop

```diff
@@ -67,6 +67,7 @@ Expr60: Box<Expression> = {
 Expr70: Box<Expression> = {
     <subject: Expr70> <index: Index> => Box::new(Expression::IndexOperation{subject, index}),
     <subject: Expr70> "." <ident: Identifier>  => Box::new(Expression::DotOperation{subject, ident}),
+    <operation: Op60> <right: Expr80> => Box::new(Expression::UnaryOperation { operation, right }),
```

Comment:
Should `.` and `[` bind tighter than `!`? [Playground with the jexl-js](https://czosel.github.io/jexl-playground/#/?context=%7B%0A%20%20obj%3A%20%7B%20foo%3A%20false%20%7D%2C%0A%20%20array%3A%20%5Bfalse%5D%0A%7D&input=!obj.foo%20%26%26%20!array%5B0%5D).

I couldn't see a test, and I'm not familiar enough with `lalrpop` know. Please could you add tests verifying the following work:

```
foo == [false]
!foo[0] == true
```

```
foo == { "bar": false }
!foo.bar  == true
```

Reply:
Ok, I've updated my patch fixing priorities, naming, adding the tests you suggested.